src/geometry/manifold/ManifoldGeometry.h

```diff
@@ -1,9 +1,9 @@
 // Portions of this file are Copyright 2023 Google LLC, and licensed under GPL2+. See COPYING.
 #pragma once
 
-#include "Geometry.h"
+#include "geometry/Geometry.h"
 #include <glm/glm.hpp>
-#include "linalg.h"
+#include "geometry/linalg.h"
 #include "manifold/manifold.h"
```

Comment:
We should probably use angle brackets for manifold too

Reply:
Agreed. Done.